DigitalLearningSolutions.Data.Tests/Helpers/SectionContentTestHelper.cs

```diff
@@ -34,5 +34,15 @@ public void UpdateDiagAttempts(int tutorialId, int progressId, int diagAttempts)
                 new { tutorialId, progressId, diagAttempts }
             );
         }
+
+        public void UpdateDiagScore(int tutorialId, int progressId, int score)
```

Comment:
Please avoid abbreviations 🙂

Reply:
The other method in this helper is titled "UpdateDiagAttempts" as it is updating the diagAttempts column. Woud you prefer I change this to "UpdateDiagLast" for consistency or "UpdateDiagnosticScore" for readability in the test? (I can also change "UpdateDiagAttempts" to "UpdateDiagnosticAttempts" if that's preferred too)

Reply:
Change to `UpdateDiagnosticScore` and `UpdateDiagnosticAttempts` please. I think we should try to avoid referring to the names in the db and they can often be confusing 🙂